lte/gateway/c/core/oai/patches/CMake-warnings-not-errors.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lte/gateway/c/core/oai/oai_mme/CMakeLists.txt b/lte/gateway/c/core/oai/oai_mme/CMakeLists.txt
+index f9f6aa7ff..95b7d9fe1 100644
+--- a/lte/gateway/c/core/oai/oai_mme/CMakeLists.txt
++++ b/lte/gateway/c/core/oai/oai_mme/CMakeLists.txt
+@@ -49,7 +49,7 @@ target_link_libraries(mme
+     -Wl,--end-group
+     ${LFDS} pthread m sctp  rt crypt ${CRYPTO_LIBRARIES} ${OPENSSL_LIBRARIES}
+     ${NETTLE_LIBRARIES} ${CONFIG_LIBRARIES} gnutls ${SERVICE303_LIB}
+-    prometheus-cpp grpc grpc++ yaml-cpp
++    prometheus-cpp grpc grpc++ gpr grpc yaml-cpp yaml z cares boost_program_options boost_filesystem boost_system boost_regex double-conversion address_sorting gflags unwind iberty dl
+ )
+ 
```

Comment:
what is this patch for?

Reply:
@Raphael Defosseux (OSA)  added the above patch PR #5438

Reply:
@ganeshg87 @ulaskozat 

This patch is no longer present on `master`. I got rid of it a while ago.

I make me very worried that you are bring bad stuff to `master`....

Regards
Raphael

Reply:
@rdefosse, I am doing merge activity from magma-5G to master.
please have a look at your piece of code and comment if anything wrong.?

Reply:
I removed the above patch, as per your comment.